home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/cooperations/contacts/CooperationContactApiIT.java

```diff
@@ -1,38 +1,24 @@
 package com.softserveinc.ita.homeproject.api.tests.cooperations.contacts;
 
-import static com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil.BAD_REQUEST;
-import static com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil.NOT_FOUND;
-import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
-import static org.junit.jupiter.api.Assertions.*;
-
-import javax.ws.rs.core.Response;
-import java.math.BigDecimal;
-import java.util.ArrayList;
-import java.util.List;
-
 import com.softserveinc.ita.homeproject.ApiException;
 import com.softserveinc.ita.homeproject.ApiResponse;
 import com.softserveinc.ita.homeproject.api.CooperationApi;
 import com.softserveinc.ita.homeproject.api.CooperationContactApi;
-import com.softserveinc.ita.homeproject.api.tests.query.CooperationContactQuery;
 import com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil;
-import com.softserveinc.ita.homeproject.model.CreateContact;
-import com.softserveinc.ita.homeproject.model.Address;
-import com.softserveinc.ita.homeproject.model.ContactType;
-import com.softserveinc.ita.homeproject.model.CreateCooperation;
-import com.softserveinc.ita.homeproject.model.CreateEmailContact;
-import com.softserveinc.ita.homeproject.model.CreatePhoneContact;
-import com.softserveinc.ita.homeproject.model.CreateHouse;
-import com.softserveinc.ita.homeproject.model.ReadContact;
-import com.softserveinc.ita.homeproject.model.ReadEmailContact;
-import com.softserveinc.ita.homeproject.model.ReadPhoneContact;
-import com.softserveinc.ita.homeproject.model.UpdateContact;
-import com.softserveinc.ita.homeproject.model.UpdateEmailContact;
-import com.softserveinc.ita.homeproject.model.UpdatePhoneContact;
-import com.softserveinc.ita.homeproject.model.ReadCooperation;
+import com.softserveinc.ita.homeproject.model.*;
```

Comment:
Please try to avoid using wildcard imports. Full import increase readability on the review stage.

Reply:
Okay